app/models/atheneum/command/upsert_exam.rb

```diff
@@ -22,4 +23,10 @@ def self.parse_json(exam_json)
     ['start_time', 'end_time'].each { |param| exam[param] = exam[param].to_time }
     exam
   end
+
+  def self.validate_repeated_exam(id, guide_id)
+    if Exam.find_by(classroom_id: id).blank? && Exam.find_by(guide_id: guide_id)
+      Exam.find_by(guide_id: guide_id).destroy!
+    end
```

Comment:
You could use the `exists` method instead of `Exam.find_by(classroom_id: id).blank?`